### DISCUSSION TITLE
Cli tool not working

### DISCUSSION THREAD

**Discussion by STashakkori** [Byron/google-apis-rs#305] [Q&A] [answered]
When I run translate2, I get the following message:

Please direct your browser to https://accounts.google.com/o/oauth2/auth?scope=https://www.googleapis.com/auth/cloud-platform&access_type=offline&redirect_uri=http://127.0.0.1:36277&response_type=code&client_id=620010449518-9ngf7o4dhs0dka470npqvor6dc5lqb9b.apps.googleusercontent.com and follow the instructions displayed there.****

**Comment by Byron** [comment, marked as answer]
And there is even more to this. When the link is actually followed, the app is linked to the account used to login, but still _may_ not have permission to use the API as that requires the involvement of the google developer console.

Read more about [authentication in the docs](http://byron.github.io/google-apis-rs/google_translate3_cli/index.html).
